ArjunaCore/arjuna/tests/classes/com/hp/mwtests/ts/arjuna/tools/ExposeAllLogsTest.java

```diff
@@ -126,17 +125,13 @@ private void test(boolean exposeAllLogsViaJMX) throws Exception
 
     private void probeObjectStore(boolean exposeAllLogs, boolean useJMX) throws MBeanException {
         ObjectStoreEnvironmentBean osEnvBean = BeanPopulator.getDefaultInstance(ObjectStoreEnvironmentBean.class);
-
```

Comment:
You reformatted the code but now there is no separation between declarations and code and between control statements.

Reply:
Ok, I will return the lime back to as they were defined.